spring-webflux/src/main/java/org/springframework/web/reactive/function/client/WebClient.java

```diff
@@ -51,8 +51,8 @@
  * // Perform requests...
  * Mono&#060;String&#062; result = client.get()
  *     .uri("/foo")
- *     .exchange()
- *     .then(response -> response.bodyToMono(String.class));
+ *     .retrieve()
```

Comment:
@marcingrzejszczak note the `exchange()` is still valid but then the example should use `flatMap(response -> ...)` rather than `then`

Reply:
yeah, it's actually written later in the javadocs. I guess in this example we want the simplest case so I've picked the `retrieve` one. Does it make sense or should I change it?